src/faq.mdx

```diff
@@ -1,5 +1,70 @@
 *Note: This article is written by past USACO Finalists and is not an official FAQ.*
 
-## Q: What is USACO?
+## Q: What language should I use for USACO?
 
-lorem ipsum dolor sit amet
\ No newline at end of file
+In general, we recommend the following:
+
+- **If you already know one of these languages, just use it.**
+  - If you know multiple languages, we recommend you pick C++ over Java, and Java over Python.
+- **For Bronze, any language will do.** It is possible to receive full credit with C++, Java, and Python in Bronze.
+- **For Silver<Asterisk>However, if Python is the only language you know / you really want to use Python, you can still pass Silver with Python.</Asterisk>, Gold, and Platinum, Python is not recommended**, because it is a slow language and lacks an ordered map.
+
+Don't overthink choosing a language -- you can always change languages later!
+
+We cover choosing a language in more detail in our ["Choosing a Language"](https://usaco.guide/general/choosing-lang) module.
+
+## Q: How do I prepare for USACO?
+
+Learn algorithms, do practice problems, and reflect on why you're missing problems. Make sure you learn from every problem you do, and you'll improve over time. If you're looking for a guided roadmap to improve at USACO, check out the [USACO Guide](https://usaco.guide/) (that's this site!).
+
+If you want to get better at USACO, the key thing is to **do more practice**!
+
+## Q: What's the best resource to get better at USACO?
+
+We made the [USACO Guide](https://usaco.guide/) specifically to provide high-quality resources to help people get better at USACO; we encourage you to give it a shot! We also list [additional USACO resources](https://usaco.guide/general/resources-cp) that you may find helpful. Additionally, USACO.org has its own [resources page](http://usaco.org/index.php?page=resources).
+
+## Q: I'm stuck. Where can I get help for USACO?
+
+We recommend you go to the [(unofficial) USACO Forum](https://forum.usaco.guide/) to get help when you're stuck. Alternatively, you can join the [(unofficial) USACO Discord Server](https://discord.com/invite/bessMBe). They have channels called #cp-discussion and #cp-help specifically dedicated to questions about competitive programming.
+
+## Q: How does USACO contest scoring work?
+
+USACO Contests are scored out of 1000 points. Each contest typically has 3 problems (sometimes 4); each problem is weighted equally, and is typically worth $1000 / 3 = 333$ points. Each problem then has a number of test cases (typically 10), and each test case is weighted equally. For example, if a contest has three problems, and problem A has 10 test cases, then you will get $333/10 = 33$ points for each test case that you get correct for problem A.
```

Comment:
Are there really typically 10 TCs? I think the gold/plat contests have a lot more these days